packages/trip-details/src/index.tsx

```diff
@@ -91,6 +91,17 @@ const TransitFare = ({
 }: TransitFareProps): ReactElement => {
   const currentFare = transitFares[fareKey];
 
+  // TODO: Is this needed? Every implementation of TransitFare does a check for currentFare's existence, although not the cents field
+  if (!currentFare?.cents) {
```

Comment:
Does that include cases where `cents === 0`?

Reply:
Oh very good catch thank you! Fixed in 26b50fef415e1dbf0339c25a520b56697861532b